template.md

```diff
@@ -0,0 +1,305 @@
+---
+# required metadata
+
+title: [ARTICLE TITLE | SERVICE NAME]
+description:
+keywords:
+author: [GITHUB USERNAME]
+manager: [MANAGER ALIAS]
```

Comment:
I thought about putting wpickett as the default here to make it easier. I think it'd be better to keep track of issues popping up in our BI dashboard than having dotnet-bot as the manager.